src/browser/modules/EditorInfo/EditorInfo.jsx

```diff
@@ -6,16 +6,32 @@ import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import { withBus } from 'react-suber'
 import { EditNodes } from './EditNodes'
+import {
+  Drawer,
+  DrawerHeader,
+  DrawerBody,
+  DrawerSection
+} from 'browser-components/drawer/index'
+import { EntityType } from './EntityType'
 
 export class EditorInfo extends Component {
   render () {
-    return (
+    return this.props.selectedItem ? (
       <div>
         <EditNodes
           nodeProperties={this.props.selectedItem._fields[0].properties}
           entityType={this.props.entityType}
         />
       </div>
+    ) : (
+      <Drawer id='db-drawer'>
+        <DrawerHeader>Editor</DrawerHeader>
+        <DrawerBody>
+          <DrawerSection>
+            <EntityType itemType='Canvas' />
```

Comment:
replace entity type with message about no selection. try with english which lures user to select either node or relationship

Reply:
done with message for the user